projects/high_order_cell_assembly/analyze_spencer_data.m

```diff
@@ -36,10 +36,11 @@
 % %         load dataToSubutai_102616/Combo3_V1.mat
 % end
 
-load('/Users/ycui/Documents/SpencerData/data/Combo3_V1andAL.mat')
-spiketrain = Combo3_V1andAL(3, 1).spiketrain;
+load('./data/Combo3_V1andAL.mat')
```

Comment:
For each of these files, please add comment in the top stating the purpose of the file.